src/index.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-loop-func */
+import './style.css';
+import theCheck from './modules/checkbox.js';
+
+let toDoList = [];
+
+function CreateTask(description, completed, id) {
+  this.description = description;
+  this.completed = completed;
+  this.id = id;
+}
```

Comment:
I suggest that you convert this function into a class so as to follow ES6 syntax